extensions/wikia/AbTesting/js/AbTest.js

```diff
@@ -207,6 +207,9 @@
 	// Returns true if value falls into provided ranges
 	function isInRanges( value, ranges ) {
 		var i, range;
+		if ( !ranges || !ranges.length ) {
```

Comment:
Object.isArray is available in IE9+, I wouldn't like to intentionally break it for IE8-
`ranges.length` check was supposed to emulate isArray.